### ISSUE TITLE
Unable to provide plugin options via flakehell

### ISSUE THREAD

**Issue by bitphage** [flakehell/flakehell#7]
Let's say I have `flake8-keyword-arguments` and want to pass `--max-pos-args=3` option. With flake8, I just adding `max-pos-args = 3` to `.flake8` or passing `--max-pos-args=3`. Neither works for flakehell.

Tried to put into pyproject.toml, neither works as well:
```
[tool.flakehell]
max_pos_args = 3
max-pos-args = 3
```

**Comment by bitphage** [comment]
Grrr... traced this down to `_checkers.py.FlakeHellCheckersManager.make_checkers` and `Snapshot` class to find out that it's just uses cache. Manual cache removal did the trick. `rm -rf ~/.cache/flakehell`

**Comment by bitphage** [comment]
Probably flakehell should be intelligent enough to detect option changes...

**Comment by howeaj** [comment]
Ran into the same problem with pylint options.

Some kind of --force or --clear-cache option would also help.

**Comment by nobbynobbs** [comment]
Yeah! This behavior cost me couple of hours the day I've started using flakehell 😆
